db_master_deploy/deploy.sh

```diff
@@ -422,6 +469,12 @@ done
 
 attached_slaves=$(psql -qAt -h localhost -d postgres -c "SELECT count(1) from pg_stat_replication where state IN ('streaming') and client_addr::text ~* '${PUBLISHED_SLAVES}';")
 
+# manual deploy will be creating a lock file in tmp/ directory for each deploy source
+if [[ ${comment} == "manual db deploy"  ]]; then
+    check_locks
+    trap clean_locks SIGHUP SIGINT SIGTERM INT TERM EXIT
```

Comment:
Ok nice, so we trigger clean_locks on any of these signal. What about `SIGQUIT` ctrl+D